src/haz3lweb/LangDocMessages.re

```diff
@@ -1586,6 +1621,47 @@ let conapp_exp: form = {
     examples: [conapp_exp_ex],
   };
 };
+let _exp_fun = exp("e_fun");
+let _exp_arg = exp("e1");
+let _exp_deferral = deferral();
+let deferred_funapp_exp_coloring_ids =
+    (~x_id: Id.t, ~supplied_id: Id.t, ~deferred_id: Id.t)
+    : list((Id.t, Id.t)) => [
+  (Piece.id(_exp_fun), x_id),
+  (Piece.id(_exp_arg), supplied_id),
+  (Piece.id(_exp_deferral), deferred_id),
+];
+let deferred_funapp_exp: form = {
+  let explanation = {
+    message: "Partial function application. Apply the [*function*](%i) to the [*supplied arguments*](%i). The [*deferred arguments*](%i) can be applied in future applications.",
+    feedback: Unselected,
+  };
+  let comma = comma_exp();
+  let or_exp = () =>
+    Example.mk_tile(Form.mk_infix("|", Rul, Precedence.or_), []);
```

Comment:
this is rendering as red (sort inconsistency error) as this is not a valid hazel operator on its own... This needs to be a concrete example, not a schema... probably just drop the "e1 |" part and just go with the "_"